task/project/app/models.py

```diff
@@ -14,7 +14,8 @@
 def get_file_path(instance, filename):
     ext = filename.split('.')[-1]
     filename = "%s.%s" % (uuid.uuid4(), ext)
-    return os.path.join('', filename)
+    path = os.path.join('', filename)
+    return context['request'].build_absolute_uri(obj.profile_img)
```

Comment:
It is storing the file path with backend host, What if due to some issue we need to change the backend host ?